src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        Random random = new Random();
+        ball.setNumber(random.nextInt(100));
+        ColorSupplier supplier = new ColorSupplier();
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/63095184/769b719b-c98c-4e0d-ac9d-3aa8125d6076)
Random and ColorSup.